src/Autonomous.cpp

```diff
@@ -142,34 +135,32 @@ void Autonomous::autonomyIter()
 	{
 		std::cout << "arrived at gate" << std::endl;
 		std::cout << "x: " << pose(0) << " y: " << pose(1) << " theta: " << pose(2)
-				  << std::endl;
+					<< std::endl;
 		landmarkFilter.reset(); // clear the cached data points
 		setCmdVel(0, 0);
 	}
 	else
 	{
-		PointXY driveTarget = getTarget();
-		if (dist(driveTarget, point_tToPointXY(pose)) < 25)
+		pose_t driveTarget = getTarget();
+
+		// if we have some existing data or new data, set the target using the landmark
+		// data
+		bool landmarkVisible = leftPostLandmark[2] != 0;
+		if (landmarkFilter.getSize() > 0 || landmarkVisible)
 		{
-			// if we have some existing data or new data, set the target using the landmark
-			// data
-			bool landmarkVisible = leftPostLandmark[2] != 0;
-			if (landmarkFilter.getSize() > 0 || landmarkVisible)
+			if (!landmarkVisible)
 			{
-				if (!landmarkVisible)
-				{
-					// we have no new data, so use the data already in the filter
-					driveTarget = point_tToPointXY(landmarkFilter.get());
-				}
-				else
-				{
-					// transform and add the new data to the filter
-					transform_t invTransform = toTransform(pose).inverse();
-					point_t landmarkMapSpace = invTransform * leftPostLandmark;
-					// the filtering is done on the target in map space to reduce any phase lag
-					// caused by filtering
-					driveTarget = point_tToPointXY(landmarkFilter.get(landmarkMapSpace));
-				}
+				// we have no new data, so use the data already in the filter
+				driveTarget = landmarkFilter.get();
+			}
+			else
+			{
+				// transform and add the new data to the filter
+				transform_t invTransform = toTransform(pose).inverse();
+				point_t landmarkMapSpace = invTransform * leftPostLandmark;
+				// the filtering is done on the target in map space to reduce any phase lag
+				// caused by filtering
+				driveTarget = landmarkFilter.get(landmarkMapSpace);
```

Comment:
The `landmarkFilter` filters on `point_t`s, which means it has data of the form `[x, y, 1]`. However, `driveTarget` is a `pose_t`, which means it has data of the form `[x, y, theta]`. If we set output of the filter to the drive target, then the target theta will be misinterpreted to be 1rad, which may not be the case.

Granted, the theta value in `driveTarget` is never used in any calculations so it doesn't affect anything right now, so I'll defer to your judgement on how to fix this.